CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION  3.10)
+project(SheenBidi)
+
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/Headers)
+file(GLOB SRC_FILES Source/*.c)
+
+option(BUILD_SHARED_LIBS "Build shared library" OFF)
+
+if(BUILD_SHARED_LIBS)
+set(BUILDSHARE SHARED)
+else()
+set(BUILDSHARE )
+endif()
+
+add_library(sheenbidi ${BUILDSHARE} ${SRC_FILES})
```

Comment:
This is highly unusual for CMake. Whether libraries are built as shared is controlled by the built-in `BUILD_SHARED_LIBS` CMake option. There is no need for and there shouldn't be such an option.